### DISCUSSION TITLE
Warp Drive for Space Stations

### DISCUSSION THREAD

**Discussion by CLtheman1** [TeamGalacticraft/Galacticraft#242] [Feature Requests]
[Galacticraft 5] Warp Drive for space stations. This crafting recipe could combine the orion drive with ender pearls, chorus fruit (or materials derived from those) to harness their space warping powers. This could be crafted into a warp core and warp engines. Control wires(or info wires) should be added to connect the warp core with the warp engines and a control interface block. These new wires should also have a sealable block. When using the control interface, it should show how much power is needed and how much power is supplied, and a button to bring up the map. Instead of destinations being based on tiers, they should be available based on how much power the warp core/engines are supplied with.

Visual effects: When the station is going into warp, the skybox should rotate and the station should point to its destination as the biome changes from "(planetname)_orbit" to "space" where the celestial bodies are large models that are far apart, and the station should move to its destination, where it'll make an orbit dimension for that planet/moon. For interstellar warp, it should have the same visual effect, but with a moving starfield.

Montage's "Dynamic Stations" suggestion would fit nicely with this.

**Comment by ghost** [comment]
This sounds like a very cool idea.
